examples/darwin-framework-tool/commands/common/MTRError.mm

```diff
@@ -26,11 +26,11 @@
 #import <lib/support/TypeTraits.h>
 
 // Stolen for now from the framework, need to export this properly.
-@interface MTRErrorHolder : NSObject
+@interface DFTErrorHolder : NSObject
```

Comment:
should the mm file be `DFTErrorHolder.mm` then?

Reply:
Not really. This is an interface that lives only into the file and is not exposed to the outside world.